src/tdymaterialproperties.c

```diff
@@ -37,26 +43,78 @@ PetscErrorCode MaterialPropertiesDestroy(MaterialProp *matprop){
   PetscFunctionReturn(0);
 }
 
+//
+// Are material properties set
+//
+
+/* -------------------------------------------------------------------------- */
+/// Reports if permeability is set
```

Comment:
Does this refer to whether a material property has been set either by a command-line argument or by a function call in the program? If so, might be worth mentioning here.

Reply:
Ah, I see, the flags are set only by calls to the appropriate functions.